policyengine_us/variables/gov/states/az/tax/income/credits/property_tax_credit/az_property_tax_credit_eligible.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class az_property_tax_credit_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Arizona Property Tax Credit"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.az.tax.income.credits.property_tax
+
+        age = person("age", period)
+        age_eligible = age >= p.age_threshold
+
+        ssi = person("ssi", period)
+        ssi_eligible = ssi > 0
+        age_or_ssi_eligible = tax_unit.any(age_eligible | ssi_eligible)
```

Comment:
do elderly or SSI-receiving dependents qualify for it? the legal code suggests not